### PR TITLE
Remove old NarrativeManager.js

### PR DESCRIPTION
**Pull request by briehl**
# Description of PR purpose/changes

The functionality in `kbase-extension/static/kbase/js/api/NarrativeManager.js` has long since been moved to remote NarrativeService calls. @dauglyon encountered this earlier, and I was reminded that it wasn't actually removed from this repo's codebase. Oops.

This PR fixes that by removing the module and dangling references to it. It's only imported in one module, and there it's just initialized and never used.

# Testing Instructions
* Details for how to test the PR: 
- [x] Tests pass in travis and locally 
- [x] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have run Black and Flake8 on changed Python Code manually or with git precommit (and the travis build passes)

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -103,7 +103,7 @@
         "google_ad_conversion": "kR9OCLas4JgBEOy2pucC"
     },
     "comm_wait_timeout": 600000,
-    "config": "narrative-refactor",
+    "config": "dev",
```
Comment (ialarmedalien) [inline src/config.json:106]: deliberate?

Comment (briehl) [reply src/config.json:106]: Oh. Forgot about that.

Uh, semi-deliberate. It doesn't affect anything except for a dev downloading the module. Running locally against narrative-refactor doesn't seem to work for me.

**Review by ialarmedalien — APPROVED** [review]
LGTM

**Comment by briehl** [comment]
Once Travis testing passes, I'll merge.

**Pull request merged**